sys/include/net/nanocoap.h

```diff
@@ -157,17 +157,9 @@ extern "C" {
  * @brief    Maximum length of a CoAP header for a blockwise message
  */
 #ifndef CONFIG_NANOCOAP_BLOCK_HEADER_MAX
-#define CONFIG_NANOCOAP_BLOCK_HEADER_MAX   (64)
+#define CONFIG_NANOCOAP_BLOCK_HEADER_MAX   (80)
```

Comment:
Hmm, is there some reasoning behind this number? If so, it would be helpful to add a short comment explaining that reasoning.

Reply:
Previously the work buffer was also used for the request.
Now that we don't need a work buffer anymore, we allocate the request header on the stack.
Turns out it needs to be a bit larger for e.g. SUIT. (previously this define was not used for enforcing a length restriction, but to calculate the requires user buffer size by doing `CONFIG_NANOCOAP_BLOCK_HEADER_MAX + BLOCKSIZE`.

Reply:
So maybe add

```
/* Value obtained experimentally when using SUIT */
```

in front, so that other developers how to check for regressions when changing this?